core/src/main/scala/dagr/core/cmdline/parsing/ParsingUtil.scala

```diff
@@ -35,15 +36,17 @@ import scala.collection.immutable.Map
 import scala.collection.mutable.ListBuffer
 
 /** Variables and Methods to support command line parsing */
-private[parsing] object ParsingUtil {
+private[parsing] object ParsingUtil extends Configuration  {
   /** Gets the [[CLPAnnotation]] annotation on this class */
   def findClpAnnotation(clazz: Class[_]): Option[CLPAnnotation] = {
     ReflectionUtil.findJavaAnnotation(clazz, classOf[CLPAnnotation])
   }
 
+  private lazy val printColor: Boolean = optionallyConfigure[Boolean](Configuration.Keys.ColorStatus).getOrElse(false)
```

Comment:
Two things here:
1) Perhaps we should just add this to the reference.conf and not use a default here?
2) If you want to keep the default in code there's also `configuration[Boolean](Configuration.Keys.ColorStatus, false)`